### PR TITLE
One more pass on null rod sprites + null rod storage sanity pass

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

Makes several null rod variants match up with their sprites in wearable locations, as well as finally cleaning up the remaining sprites for both nullrods and bibles.

Also includes some resprites for a few inhands.

I got really lazy at the end of this, please don't blame me for the shitty pitchfork back sprite okay?

## Why It's Good For The Game

[I don't want pink sprites anymore. I just want to be happy.](https://www.youtube.com/watch?v=vZa0Yh6e7dw)

## Changelog
:cl:
fix: Fixes the vast majority of the problems with null rod sprites and bible sprites.
fix: Null rods storage spots properly align with worn icon sprites. This includes some size adjustments. This is basically meaningless except for storage.
imageadd: A sprite for the pride hammer in-hand.
imageadd: New inhands for the moonlight greatsword and extradimensional sword.
/:cl:

### CODE REVIEW

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -200,6 +199,7 @@
 	throw_speed = 3
 	throw_range = 4
 	throwforce = 10
+	slot_flags = ITEM_SLOT_BELT
```
Comment (ATH1909) [inline code/game/objects/items/holy_weapons.dm:202]: finally

File: code/game/objects/items/holy_weapons.dm [hunk 2]
```diff
@@ -311,12 +311,12 @@
...
 	icon_state = "claymore_gold"
 	inhand_icon_state = "claymore_gold"
-	worn_icon_state = "claymore"
+	worn_icon_state = "claymore_gold"
 	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
 	name = "holy claymore"
 	desc = "A weapon fit for a crusade!"
-	w_class = WEIGHT_CLASS_HUGE
+	w_class = WEIGHT_CLASS_BULKY
```
Comment (ATH1909) [inline code/game/objects/items/holy_weapons.dm:319]: will this interfere with storing these in your chaplain armor's suit storage slot?

File: code/game/objects/items/holy_weapons.dm [hunk 3]
```diff
@@ -359,22 +360,21 @@
...
-	slot_flags = ITEM_SLOT_BELT | ITEM_SLOT_BACK
 
 /obj/item/nullrod/claymore/multiverse
 	name = "extradimensional blade"
 	desc = "Once the harbinger of an interdimensional war, its sharpness fluctuates wildly."
 	icon_state = "multiverse"
 	inhand_icon_state = "multiverse"
-	slot_flags = ITEM_SLOT_BELT
+	worn_icon_state = "multiverse"
+	slot_flags = ITEM_SLOT_BACK
```
Comment (ATH1909) [inline code/game/objects/items/holy_weapons.dm:377]: 😕

File: code/game/objects/items/holy_weapons.dm [hunk 4]
```diff
@@ -386,31 +386,35 @@
 	icon = 'icons/obj/transforming_energy.dmi'
 	icon_state = "swordblue"
```
Comment (ATH1909) [inline code/game/objects/items/holy_weapons.dm:387]: at some point, we really oughtta let people turn these things on and off.

but that's for another PR, I suppose.

File: code/game/objects/items/holy_weapons.dm [hunk 5]
```diff
@@ -386,31 +386,35 @@
...
+	worn_icon_state = "swordblue"
 	slot_flags = ITEM_SLOT_BELT
+	desc = "If you strike me down, I shall become more robust than you can possibly imagine."
 
 /obj/item/nullrod/claymore/saber/red
 	name = "dark energy sword"
 	icon_state = "swordred"
 	inhand_icon_state = "swordred"
+	worn_icon_state = "swordred"
 	desc = "Woefully ineffective when used on steep terrain."
```
Comment (ATH1909) [inline code/game/objects/items/holy_weapons.dm:398]: while we're in the area, I'm kind of wondering if "on" (instead of "in") is the right word to use here

also, it'd be absolutely hilarious if we added some sort of multi-z mechanic effect to this description, but again, that'd be for another PR.

File: code/game/objects/items/holy_weapons.dm [hunk 6]
```diff
@@ -386,31 +386,35 @@
...
 	desc = "This thing is so unspeakably HOLY you are having a hard time even holding it."
 	icon_state = "sord"
 	inhand_icon_state = "sord"
+	worn_icon_state = "sord"
 	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
-	slot_flags = ITEM_SLOT_BELT
 	force = 4.13
 	throwforce = 1
+	slot_flags = ITEM_SLOT_BELT
```
Comment (ATH1909) [inline code/game/objects/items/holy_weapons.dm:417]: ~~remove the item_slot_belt flag to make it even shittier~~

File: code/game/objects/items/holy_weapons.dm [hunk 7]
```diff
@@ -386,31 +386,35 @@
...
 /obj/item/nullrod/claymore/saber/pirate
 	name = "nautical energy sword"
 	icon_state = "cutlass1"
 	inhand_icon_state = "cutlass1"
+	worn_icon_state = "swordred"
 	desc = "Convincing HR that your religion involved piracy was no mean feat."
 
 /obj/item/nullrod/sord
 	name = "\improper UNREAL SORD"
 	desc = "This thing is so unspeakably HOLY you are having a hard time even holding it."
```
Comment (ATH1909) [inline code/game/objects/items/holy_weapons.dm:409]: I assume that the lack of "that" between "HOLY" and "you" is a reference, yes?

**Comment by ATH1909** [comment]
good stuff right here, thanks

File: code/game/objects/items/holy_weapons.dm (hunk 2 above)
Comment (necromanceranne) [reply code/game/objects/items/holy_weapons.dm:319]: This actually allows you to place it there in the first place.

File: code/game/objects/items/holy_weapons.dm (hunk 4 above)
Comment (necromanceranne) [reply code/game/objects/items/holy_weapons.dm:387]: Transforming component.

File: code/game/objects/items/holy_weapons.dm (hunk 7 above)
Comment (necromanceranne) [reply code/game/objects/items/holy_weapons.dm:409]: Yeah it's I believe a direct quote except SHITTY (or whatever term was used I forget) was replaced with HOLY. It parses like Hussie's writing at least.

**Review by Twaticus — APPROVED** [review]
